src/sprints.actions.js

```diff
@@ -1,22 +1,47 @@
 const { getTeamId } = require('./team-data');
-const client = require('./jira-client');
+const Paginator = require('./data/paginator');
+const jiraClient = require('./jira-client');
+const { makeGetRequest } = jiraClient;
 
-async function getSprints({ team }) {
+async function getSprints({ team }, query = {}) {
   const teamId = getTeamId(team);
-  const sprints = await client.makeGetRequest(`board/${ teamId }/sprint`);
   const summary = await getVelocities(teamId);
   const velocities = summary.velocityStatEntries;
-  return sprints.values
-    .filter(sprint => sprint.originBoardId === teamId)
-    .sort((a, b) => b.id - a.id)
-    .map(sprint => Object.assign({}, sprint, {
-	    velocity: velocities[sprint.id] ? velocities[sprint.id].completed.value : 0,
-	    estimated: velocities[sprint.id] ? velocities[sprint.id].estimated.value : 0
-    }));
+
+  const paginator = new Paginator({
+    async fetchPage(query) {
+      // TODO (indexzero): support other query string params (startAt, maxResults)
+      // See: https://docs.atlassian.com/jira-software/REST/7.0.4/#agile/1.0/board/{boardId}/sprint
+
+      // TODO (indexzero): make state configurable)
```

Comment:
when will these TODOs be addressed?

Reply:
In the undecided future?

Reply:
no concrete plans. hoping to address as I continue to work on this repo w/ @indexzero